docs/modules/ROOT/pages/documentation/extensions/grpc/README.adoc

```diff
@@ -0,0 +1,449 @@
+[[grpc]]
+= gRPC
+
+[[what-is-grpc]]
+== What is gRPC?
+
+gRPC is a modern open-source high performance Remote Procedure Call (RPC) framework that use HTTP2 and simplifies the communication with protocol buffers to share data between client and server. By enabling gRPC Payara Server gives more options to create binary adaptable services.
+
+[[what-can-we-do-with-it]]
+== What can we do with it?
+
+* We can create services more efficient than REST.
+* Use HTTP/2 to provide full duplex communication between the client and the service.
+* Use protocol buffers to optimize and simplify communication.
+* Enables bidirectional streaming out-of-the-box.
+
+gRPC supports the means for users to implement gRPC style service programming.
+
+[[installing-grpc-server-support-module]]
+== Installing gRPC Server Support Module
+
+You need to download the following artifact from enterprise nexus repository: https://nexus.payara.fish/repository/payara-enterprise .
+
+NOTE: you must have enterprise credentials to download the artifact.
```

Comment:
Rephrase this admonition as: 

IMPORTANT: You must have an active Payara Enterprise subscription to access this Nexus repository.

Reply:
already done